src/app/weightings/weightings.component.html

```diff
@@ -0,0 +1,230 @@
+
+
+<nav class="navbar navbar-default" style="color:white;">
+  <div class="container-fluid">
+    <div class="navbar-header"  >
+      <img src="../../../assets/images/logo.png" alt="Logo" style ="width: 50px; height: 50px;">
+       <a id="site-head" class="navbar-brand" style="color:white;" href="#">Saffron </a>
+   </div>
+
+	 
+	 <div class="container "style="color:white;"> 
+
+
+		 <div class="col-md-2 text-right pull-right" >
+			 <span class="pull-right">
+				<span class="glyphicon glyphicon-user"></span>
+			 </span>
+			 <p id='studentDeets'>Student Name uXXXXXXXX<p>
+     </div>
+       <ul class="nav nav-pills" >
+        <li class="active"><a id="li" href="Grading" style="background-color:#17294D;color:white;">Manage Marks</a></li>
+        <li ><a id="li" href="Assignment" style="color:white;">Manage Assignments</a></li>
+        <li ><a  id="li" href="Weights" style="color:white;">Manage Weightings</a></li>
+      </ul> 
+		
+	 </div>
+  
+  </div>
+</nav>
+
+  <h2 id="li" class="center">Weightings</h2>
+<div class="tg-wrap"><table class="tg">
+  <tr>
+    <th id="li" class="tg-yw4l">Assessment</th>
+    <th id="li" class="tg-z1yq">Weighting(%)</th>
+  </tr>
+  <tr>
+    <td class="tg-yw4l">
+       <div class="form-group">
+        <select class="form-control" id="sel1">
+          <option id="li">Assignment</option>
+          <option id="li">Test</option>
+          <option id="li">Exam</option>
+      </select>
+      </div>
+    </td>
+    <td class="tg-yw4l">
+      <div class="form-group">
+        <select class="form-control" id="sel1">
+          <option>5%</option>
+          <option>10%</option>
+          <option>15%</option>
+          <option>20%</option>
+          <option>25%</option>
+          <option>30%</option>
+          <option>35%</option>
+          <option>40%</option>
+          <option>45%</option>
+          <option>50%</option>
+          <option>55%</option>
+          <option>60%</option>
+          <option>65%</option>
+          <option>70%</option>
+          <option>75%</option>
+          <option>80%</option>
+          <option>85%</option>
+          <option>90%</option>
+          <option>95%</option>
+          <option>100%</option>
+        </select>
+      </div>
+    </td>
+  </tr>
+  <tr>
```

Comment:
Why are these things repeated so many times? Is it meant to be mock data? Can it not use `ngFor` with the mock data rather so that it can automatically update on how many it should make